eventmesh-test/src/main/java/org/apache/eventmesh/http/demo/sub/service/SubService.java

```diff
@@ -38,6 +44,11 @@
     final String dcn = "FT0";
     final String subsys = "1234";
 
+    // CountDownLatch size is the same as messageSize in AsyncPublishInstance.java (Publisher)
+    private CountDownLatch countDownLatch = new CountDownLatch(AsyncPublishInstance.messageSize);
+
+    private ExecutorService executorService = Executors.newFixedThreadPool(5);
```

Comment:
Of cource it won't make a difference, but I wonder here is it necessary to use a thread pool?

Reply:
It is a best practice to use the JDK Concurrency library (ExecutorService + ThreadPool) to manage the lifecycle of Thread.

Just suggestion. I can rollback to create Thread manually. Thanks for review comment.